<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjustBC</class>
 <widget class="QDialog" name="AdjustBC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>160</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>160</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Brightness/Contrast</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Brightness</string>
       </property>
       <property name="buddy">
        <cstring>spbxB</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spbxB">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="sldrB">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Contrast</string>
       </property>
       <property name="buddy">
        <cstring>spbxC</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spbxC">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="sldrC">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>&amp;Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spbxB</tabstop>
  <tabstop>spbxC</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>sldrB</tabstop>
  <tabstop>sldrC</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sldrB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spbxB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldrC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spbxC</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spbxB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldrB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spbxC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldrC</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldrB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AdjustBC</receiver>
   <slot>adjusted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldrC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AdjustBC</receiver>
   <slot>adjusted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdjustBC</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdjustBC</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>adjusted()</slot>
 </slots>
</ui>
